### ISSUE TITLE
"Could not launch chrome"

### ISSUE THREAD

**Issue by olafure** [c9fe/22120#61]
```
$ git rev-parse HEAD 
71a33c810fcff4006405bb17ae0300f1db3a8db2

$ ls -la /opt/google/chrome/chrome
-rwxr-xr-x 1 root root 155654248 Feb 17  2020 /opt/google/chrome/chrome

$ node --version 
v14.3.0

$ npm i && npm start 
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@2.1.3 (node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@2.1.3: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@1.2.13 (node_modules/watchpack-chokidar2/node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.2.13: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})

audited 503 packages in 2.563s

7 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities


> archivist1@1.3.13 start /local/data/projects/3rd-projects/22120
> node index.js

Args usage: <server_port> <save|serve> <chrome_port> <library_path>
Running in node...
Importing dependencies...
Attempting to shut running chrome...
There was no running chrome.
Removing 22120's existing temporary browser cache if it exists...
Launching library server...
Library server started.
Waiting 1 second...
{"server_up":{"upAt":"2020-11-12T12:44:43.148Z","port":22120}}
Launching chrome...
Could not launch chrome.
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! archivist1@1.3.13 start: `node index.js`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the archivist1@1.3.13 start script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
```

**Comment by c9fe** [comment]
If you try setting DEBUG_22120 to something to get debug output on chrome launch

```console
$ export DEBUG_22120=abc
```

we can have more information to diagnose. 

It could be chrome already running. Normally we can shut the browser, but possibly it might not. You could ensure all chrome is killed before running.

If you let me know i can maybe investigate further

**Comment by olafure** [comment]
Did you intend to close this issue? I see you've closed another issue like this: https://github.com/c9fe/22120/issues/68

```
$ export DEBUG_22120=abc
$ git rev-parse HEAD 
3c88e223b24ff063ca754ae2b5b4e2c84d02782a

$ npm i && npm start 
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@2.1.3 (node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@2.1.3: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@1.2.13 (node_modules/watchpack-chokidar2/node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.2.13: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})

audited 503 packages in 2.61s

7 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities



   ╭────────────────────────────────────────────────────────────────╮
   │                                                                │
   │      New patch version of npm available! 6.14.5 → 6.14.8       │
   │   Changelog: https://github.com/npm/cli/releases/tag/v6.14.8   │
   │               Run npm install -g npm to update!                │
   │                                                                │
   ╰────────────────────────────────────────────────────────────────╯


> archivist1@1.3.13 start /local/data/projects/3rd-projects/22120
> node index.js

Args usage: <server_port> <save|serve> <chrome_port> <library_path>
Running in node...
Importing dependencies...
Attempting to shut running chrome...
There was no running chrome.
Removing 22120's existing temporary browser cache if it exists...
Launching library server...
Library server started.
Waiting 1 second...
{"server_up":{"upAt":"2020-11-14T12:54:57.302Z","port":22120}}
Launching chrome...
  ChromeLauncher No debugging port found on port 9222, launching a new Chrome. +0ms
Could not launch chrome.
Chrome launch error: ChromePathNotSetError
    at new LauncherError (/local/data/projects/3rd-projects/22120/node_modules/chrome-launcher/dist/utils.js:37:22)
    at new ChromePathNotSetError (/local/data/projects/3rd-projects/22120/node_modules/chrome-launcher/dist/utils.js:44:9)
    at Object.linux (/local/data/projects/3rd-projects/22120/node_modules/chrome-launcher/dist/chrome-finder.js:128:15)
    at Function.getFirstInstallation (/local/data/projects/3rd-projects/22120/node_modules/chrome-launcher/dist/chrome-launcher.js:131:51)
    at Launcher.<anonymous> (/local/data/projects/3rd-projects/22120/node_modules/chrome-launcher/dist/chrome-launcher.js:168:47)
    at Generator.throw (<anonymous>)
    at rejected (/local/data/projects/3rd-projects/22120/node_modules/chrome-launcher/dist/chrome-launcher.js:11:65)
    at processTicksAndRejections (internal/process/task_queues.js:97:5) {
  message: 'The environment variable CHROME_PATH must be set to executable of a build of Chromium version 54.0 or later.',
  code: 'ERR_LAUNCHER_PATH_NOT_SET'
}
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! archivist1@1.3.13 start: `node index.js`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the archivist1@1.3.13 start script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
```

**Comment by c9fe** [comment]
> Did you intend to close this issue?

@olafure I feel scared for you that you felt confused about that and maybe wasted your time not knowing what it meant
